admin/broadleaf-admin-module/src/main/java/org/broadleafcommerce/admin/web/controller/entity/AdminOfferController.java

```diff
@@ -117,12 +117,13 @@ public String addEntity(HttpServletRequest request, HttpServletResponse response
      */
     @Deprecated
     @Override
-    @RequestMapping(value = "/{id}/duplicate", method = RequestMethod.POST)
+    @RequestMapping(value = "/{id}/duplicate/{catalogId}", method = RequestMethod.POST)
```

Comment:
I think we can have a new controller in MT module that will have method with such signature